### ISSUE TITLE
Video not working?

### ISSUE THREAD

**Issue by OhSoGood** [FREEZX/fq777-954-chrome-app#3]
Hi,
The readme says the video is not working. What’s the status for it , have you identified the problem and when will you solve it? Can I help (I'm a developer and I 've a walkera wifi drone) and if so, how?

Thanks for your time and effort, and also for sharing your code by the way!

**Comment by FREEZX** [comment]
Hi,
I tried to do it but since i know very little about video decoding, i wasn't able to do it.
I managed to capture video frames to a file, and the file played well in ffplay, but i have no idea how to play them in js. I tried something with broadway.js, as people had success with it on other stuff. I successfully played the same file, but i couldn't make it accept streamed frames.

There is some useful info in this thread, but i had no time to get into it some more:
https://github.com/mbebenita/Broadway/issues/9

My attempt can be found here:
https://github.com/FREEZX/fq777-954-chrome-app/blob/master/js/main.js#L202
and 
https://github.com/FREEZX/fq777-954-chrome-app/blob/master/js/player.js

It would be great if we get video working, any help is appreciated.
